style.css

```
@@ -1,3 +1,18 @@
 * {
     font-family: 'Zen Kaku Gothic Antique', sans-serif;
+    margin-left: 5px;
+}
+
+a, #app {
```

```suggestion
body, a {
```
better to target the body tag instead of the app ID